### PR TITLE
chore/Landing-Page-update

### PR DESCRIPTION
**Pull request by ashutoshamazix**
## Description

- Updated Landing pages

## How Has This Been Tested?

- Locally

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

**Review by IAmKio — COMMENTED** [review]
LGTM but just waiting for @RanaBug's approval too

File: src/components/LandingPage/Header/index.jsx [hunk 1]
```diff
@@ -41,7 +41,7 @@ const Header = () => {
         id="header"
       >
         {/* Header Announcement */}
-        <div className="header__announcement">
+        {/* <div className="header__announcement">
```
Comment (RanaBug) [inline src/components/LandingPage/Header/index.jsx:44]: If you don't need this in the future, can it be removed please?

File: src/pages/Landing.jsx [hunk 2]
```diff
@@ -6,7 +6,7 @@ import '../styles/landing/tailwind.css';
 import { Form } from '../components/LandingPage';
 import { Footer } from '../components/LandingPage/Footer';
 import { Header } from '../components/LandingPage/Header';
-import { Popup } from '../components/LandingPage/Popup';
+// import { Popup } from '../components/LandingPage/Popup';
```
Comment (RanaBug) [inline src/pages/Landing.jsx:9]: Same here, if not needed, please remove

**Review by RanaBug — APPROVED** [review]
All good, just some comments that might need to be removed within the code if not needed in the near future

File: src/components/LandingPage/Header/index.jsx (hunk 1 above)
Comment (ashutoshamazix) [reply src/components/LandingPage/Header/index.jsx]: Removed commented-out code.

File: src/pages/Landing.jsx (hunk 2 above)
Comment (ashutoshamazix) [reply src/pages/Landing.jsx]: Removed this.

**Pull request merged**